db/README.md

```diff
@@ -0,0 +1,39 @@
+# Key-Value Database #
+
+Databases supporting mappings of arbitrary byte sequences.
```

Comment:
Can we remove all `#` suffixes from headers? Not sure why they were added.

Reply:
Yes, in Emacs I used a shortcut to insert the headers, which for some reason adds the optional trailing hashes